src/App.vue

```diff
@@ -1,26 +1,55 @@
 <template>
-  <img alt="Vue logo" src="./assets/logo.png">
-  <HelloWorld msg="Welcome to Your Vue.js App"/>
+<nav class="content">
+    <router-link to="/" class='heading'>Project</router-link>
+    <router-link to="/addnewtodo" class='heading'>
+        Add a new Project
+    </router-link>
+    <router-view></router-view>
+</nav>
 </template>
 
 <script>
-import HelloWorld from './components/HelloWorld.vue'
-
 export default {
-  name: 'App',
-  components: {
-    HelloWorld
-  }
+    name: 'App',
+    computed: {
```

Comment:
you are not using currentRoute computed anywhere in this App component, then you did you declare it? @ShivanshSinghnnovative